tools/appveyor.psm1

```diff
@@ -527,4 +541,7 @@ function Invoke-AppveyorFinish
     catch {
         Write-Host -Foreground Red $_
     }
+    finally {
+        $host.SetShouldExit($exitCode)
```

Comment:
Why do you need to request the current runspace to end?

Reply:
Because a throw statement would not make the build red. We had this problem in the past and tried to use AppVeyors cmdlets but they did not work. Given that this function is already AppVeyor specific, I think this is OK. See this link for details:
https://help.appveyor.com/discussions/problems/4498-powershell-exception-in-test_script-does-not-fail-build

Reply:
`appveyor.yml` is used in `Invoke-AppVeyorFull`. However, I don't think we can run `Invoke-AppVeyorFull` locally since AppVeyor specific cmdlets are used. So I'm fine with this change.

But it would be better we add your explanation above in the comment. I will do that shortly.

Reply:
Thanks.